### PR TITLE
feature/fix cards,shadows, corners

### PR DESCRIPTION
**Pull request by xidealo**

### CODE REVIEW

File: app/src/main/java/com/bunbeauty/papakarlo/feature/address/screen/user_address_list/UserAddressListFragment.kt [hunk 1]
```diff
@@ -51,26 +51,43 @@ class UserAddressListFragment :
...
+            UserAddressListScreen(
+                userAddressListState = UserAddressListAndroidState(
+                    userAddressItems = addressListState.userAddressList.map { userAddress ->
+                        userAddressItemMapper.toItem(userAddress)
+                    },
+                    state = addressListState.state,
+                    eventList = addressListState.eventList,
+                ),
+                onCreateAddressClick = viewModel::onCreateAddressClicked,
+                userAddressItemMapper = userAddressItemMapper
```
Comment (MaxAstin) [inline app/src/main/java/com/bunbeauty/papakarlo/feature/address/screen/user_address_list/UserAddressListFragment.kt:63]: We should't pass mappers to `Composable` functions

File: app/src/main/java/com/bunbeauty/papakarlo/feature/consumer_cart/ui/CartProductItem.kt [hunk 2]
```diff
@@ -37,17 +33,12 @@ fun CartProductItem(
...
+    FoodDeliveryCard(
         modifier = modifier
-            .card()
-            .clickable(
-                interactionSource = remember { MutableInteractionSource() },
-                indication = rememberRipple(),
-                onClick = onClick
-            ),
-        colors = FoodDeliveryTheme.colors.cardColors()
+            .card(),
```
Comment (MaxAstin) [inline app/src/main/java/com/bunbeauty/papakarlo/feature/consumer_cart/ui/CartProductItem.kt:40]: It's not a good idea to use this extension. 
1) It's not clear what this function do and you need check its implementation every time. 
2) We save only 1 line of code

**Review by MaxAstin — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/bunbeauty/papakarlo/feature/address/screen/user_address_list/UserAddressListFragment.kt (hunk 1 above)
Comment (xidealo) [reply app/src/main/java/com/bunbeauty/papakarlo/feature/address/screen/user_address_list/UserAddressListFragment.kt:63]: +

File: app/src/main/java/com/bunbeauty/papakarlo/feature/consumer_cart/ui/CartProductItem.kt (hunk 2 above)
Comment (xidealo) [reply app/src/main/java/com/bunbeauty/papakarlo/feature/consumer_cart/ui/CartProductItem.kt:40]: +

**Review by MaxAstin — APPROVED** [review]

**Pull request merged**